server/java/src/main/java/com/stripe/sample/Server.java

```diff
@@ -87,7 +88,7 @@ public static void main(String[] args) {
             // set as a query param
             SessionCreateParams.Builder builder = new SessionCreateParams.Builder()
                     .setSuccessUrl(domainUrl + "/success.html?session_id={CHECKOUT_SESSION_ID}")
-                    .setCancelUrl(domainUrl + "/canceled.html").addPaymentMethodType(PaymentMethodType.CARD)
+                    .setCancelUrl(domainUrl + "/canceled.html").addPaymentMethodType(PaymentMethodType.valueOf(paymentMethodType))
```

Comment:
For java, I think we might need to use the `addAllPaymentMethodType` method here, and also split the paymentMethodType and map to an ArrayList<string> or something? I'm not super confident about this one :)